brave-okhttp/README.md

```diff
@@ -0,0 +1,17 @@
+# brave-okhttp #
+
+Contributed by [Tristan Burch](https://github.com/tburch).
+
+This implementation makes it easy to integrate with brave to catch/trace [OkHttp](https://github.com/square/okhttp) requests. The `BraveOkHttpRequestResponseInterceptor` OkHttp [`Interceptor`](http://square.github.io/okhttp/3.x/okhttp/index.html?okhttp3/Interceptor.html) implementation will start a new Span and submit a `cs` (client sent) annotation and then submit `cr` (client received) annotation after the request is received.
+
+Example of configuring the interceptor with OkHttp:
+
+```
+import com.github.kristofa.brave.okhttp.BraveOkHttpRequestResponseInterceptor;
+
+...
+
+OkHttpClient client = new OkHttpClient.Builder()
+    .addInterceptor(new BraveOkHttpRequestResponseInterceptor())
```

Comment:
This isn't exactly accurate as it takes some arguments. Maybe put something like `..` between the parens to keep them omitted.